src/neo/Network/P2P/RemoteNode.cs

```diff
@@ -153,9 +144,25 @@ protected override void OnReceive(object message)
                 case PingPayload payload:
                     OnPingPayload(payload);
                     break;
+                case SendVersion _:
+                    OnSendVersion();
+                    break;
             }
         }
 
+        private void OnSendVersion()
+        {
+            var capabilities = new List<NodeCapability>
+            {
+                new FullNodeCapability(Blockchain.Singleton.Height)
+            };
+
+            if (LocalNode.Singleton.ListenerTcpPort > 0) capabilities.Add(new ServerCapability(NodeCapabilityType.TcpServer, (ushort)LocalNode.Singleton.ListenerTcpPort));
+            if (LocalNode.Singleton.ListenerWsPort > 0) capabilities.Add(new ServerCapability(NodeCapabilityType.WsServer, (ushort)LocalNode.Singleton.ListenerWsPort));
+
+            SendMessage(Message.Create(MessageCommand.Version, VersionPayload.Create(LocalNode.Nonce, LocalNode.UserAgent, capabilities.ToArray())));
+        }
```

Comment:
@ShawnYun I think we can move this section to `PreStart()` method. Can you test it? @ShawnYun

Reply:
ok, I will test it

Reply:
The `PreStart()` is called when an Actor is started. So it's going to be earlier than `Register`, Akka warning is also displayed.